conformanceExamples/pom.xml

```diff
@@ -0,0 +1,108 @@
+	<!--
+  ~ Copyright 2018 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+
+~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License
+at
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed
+to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific
+language governing permissions and
+  ~ limitations under the License.
+	-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>enceladus-conformanceExamples</artifactId>
+	<packaging>jar</packaging>
+	<parent>
+		<groupId>za.co.absa</groupId>
+		<artifactId>enceladus-parent</artifactId>
+		<version>0.99.0-SNAPSHOT</version>
+	</parent>
+	<dependencies>
+		<dependency>
+			<groupId>org.scala-lang</groupId>
+			<artifactId>scala-library</artifactId>
+			<version>${scala.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.scalatest</groupId>
+			<artifactId>scalatest_2.11</artifactId>
+			<version>2.2.5</version>
+			<scope>compile</scope>
+		</dependency>
+		<!-- Application -->
+		<dependency>
+			<groupId>za.co.absa</groupId>
+			<artifactId>enceladus-conformance</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>za.co.absa</groupId>
+			<artifactId>enceladus-conformance</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.spark</groupId>
+			<artifactId>spark-core_${scala.compat.version}</artifactId>
```

Comment:
Don't you inherit most of these dependencies from the parent pom? Please double check ad keep only what is necessary.

Reply:
Most are needed to run the application. But found two jackson dependencies they were not needed (one remained).

Reply:
Most are needed to run the application. But found two jackson dependencies they were not needed (one remained).